README.md

```diff
@@ -0,0 +1,99 @@
+# Budget Generator
+
+A small web app for making budgets
+
+## Description
+
+This app creates simple budgets for the user and allows the budgets to be saved or downloaded. At the moment, there
+is no ability to log in, but that feature is planned to be added in the future.
+
+## Building and Usage
+
+In order to uset the app all you have to do is make sure docker and docker compose are on your system, then pull the docker image from docker hub using the command
+
+    docker pull edisonstuart/budget-generator
+
+After you have pulled the image down you can run it using the docker-compose.main.yaml file from the repo with the command
+
+    docker-compose -f docker-compose.main.yaml up
+
+I like the run it as detached with the -d flag because that way you can stil use your terminal window. After that the application should be running on your localhost!
+
+## Motivation
+
+My motivation for making this app was to work with python and flask and learn various packages relating to web applications
+and working with a database. I also am using this app as an opportunity to learn about containerization through docker and docker-compose.

```

Comment:
@Edison-Stuart I think there are still line break issues here? The line should just keep going; it will get wrapped when it's displayed.